src/api/ApiRequest.ts

```diff
@@ -45,6 +47,10 @@ export function fetchJSON<ResponseType>(url: string, failureErrorCode: number, o
                     () => new SDKError(new Error("Failed to parse successful response JSON."), failureErrorCode)
                 );
             }
+            if (response.status === CLOUDFLARE_RATE_LIMIT_STATUS_CODE) {
+                //Map a Cloudflare rate limit response code to a special error code
+                return Future.reject(new SDKError(new Error("Request was rate limited from too many requests."), ErrorCodes.REQUEST_RATE_LIMITED));
```

Comment:
Maybe "Request was denied due to rate limiting - submit fewer requests per second to avoid."?

Reply:
Added the first part, I'm going to leave the second part off as I don't think the error text is the place to recommend solutions.